### PR TITLE
feat: [CDS-50923]:fix-step-group-template

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
added runtime field for delegate selector in step group (right drawer)
fixed issue with replace template inside step group template
removed close button in selecting stage type popup for step group template

Screenshot 2023-01-24 at 11 17 38

Screenshot 2023-01-24 at 11 17 14

#### Screenshots
Screenshot 2023-01-24 at 11 16 32

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by litvinovstas** [comment]
retrigger jest

**Comment by litvinovstas** [comment]
retrigger titlecheck

**Comment by litvinovstas** [comment]
retrigger cypress-pipeline

**Comment by litvinovstas** [comment]
retrigger jest

**Comment by litvinovstas** [comment]
retrigger coverage

File: src/modules/70-pipeline/components/PipelineInputSetForm/StepGroupInputSetForm.tsx [hunk 1]
```diff
@@ -23,9 +25,19 @@ export function StepGroupFormSetInternal(props: {
   }
 }): JSX.Element {
   const { template, allValues, values, path, formik, readonly, viewType, allowableTypes, customStepProps } = props
+  const { getString } = useStrings()
 
   return (
     <Layout.Vertical spacing="medium" padding={{ top: 'medium' }}>
+      {template?.delegateSelectors && (
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/PipelineInputSetForm/StepGroupInputSetForm.tsx:32]: Should we check for runtime value of delegateSelectors?

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**